sys/include/atomic_utils.h

```diff
@@ -383,6 +435,29 @@ static inline uint32_t atomic_fetch_add_u32(volatile uint32_t *dest,
  */
 static inline uint64_t atomic_fetch_add_u64(volatile uint64_t *dest,
                                             uint64_t summand);
+/**
+ * @brief   Atomically add a value onto a given value
+ * @param[in,out]   dest    Add @p summand onto this value atomically in-place
+ * @param[in]       summand Value to add onto @p dest
+ * @return  The value previously stored @p dest
+ *
+ * @note    This is effectively an alias of @ref atomic_fetch_add_u64,
+ *          @ref atomic_fetch_add_32, or @ref atomic_fetch_add_16 depending
```

Comment:
```suggestion
 *          @ref atomic_fetch_add_u32, or @ref atomic_fetch_add_u16 depending
```

The others have the same typo.

Reply:
There are still some more.